xml/System.Web.Security/FormsAuthentication.xml

```diff
@@ -212,7 +212,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Web.Security.FormsAuthentication.CookieDomain%2A> property value is set in the configuration file for an ASP.NET application by using the `domain` attribute of the  configuration element. The <xref:System.Web.Security.FormsAuthentication.CookieDomain%2A> property value determines the <xref:System.Web.HttpCookie.Domain%2A> that the cookie will be used for.  
+ The <xref:System.Web.Security.FormsAuthentication.CookieDomain%2A> property value is set in the configuration file for an ASP.NET application by using the `domain` attribute of the `<forms>` configuration element. The <xref:System.Web.Security.FormsAuthentication.CookieDomain%2A> property value determines the <xref:System.Web.HttpCookie.Domain%2A> that the cookie will be used for.  
```

Comment:
forms element: <https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/1d3t3c61(v%3dvs.100)>
also on line 258, 417, 460, 465, 593, 637, 779, 941, 1004, 1066, 1261, and 1464.